mos_tests/environment/os_actions.py

```diff
@@ -669,3 +669,24 @@ def force_l3_reshedule(self, router_id, new_l3_agt_id=''):
         assert(wait(
             lambda: self.neutron.list_l3_agent_hosting_routers(router_id),
             timeout_seconds=5 * 60))
+
+    def reschedule_dhcp_agent(self, net_id, controller_fqdn):
+        agent_list = self.neutron.list_agents(
+            binary='neutron-dhcp-agent')['agents']
+        agt_id_to_move_on = [agt['id'] for agt in agent_list
+                             if agt['host'] == controller_fqdn][0]
+        logger.info('Agent id to move on {0}'.format(agt_id_to_move_on))
```

Comment:
Line 'Agent id to move on 91a17d49-9b14-477f-be9c-7d841e0b1df8' in logs will not  informatively enough, IMHO